### DISCUSSION TITLE
[Feature] include user's generated USER ID on WEB User configuration UI

### DISCUSSION THREAD

**Discussion by deddc23efb** [immich-app/immich#12234] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When managing multiple users added with defaults, the storage for each user is prefixed with a user specific generated string. It would be helpful to have those strings visible in the WEB UI associated with the user config. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by radh21301** [comment]
You can do this. After you create a user, click on edit user. I would set it to the name of the user to keep things clean.

<img width="440" alt="image" src="https://github.com/user-attachments/assets/5ae8fb10-cae3-4846-9372-27edfc34ed0c">


Though I think this field could be present in the user creation UI as well. Might be easier.

User creation UI:
<img width="428" alt="image" src="https://github.com/user-attachments/assets/c11403eb-d588-4c78-b7eb-3774b75c9af4">

**Comment by deddc23efb** [comment]
Yes - I'm aware of the storage label. The storage label is not applied in the upload folder; the userid is used there. I'm debugging problems with files not being moved out the upload directory and it would be nice to see a mapping without having to examine the files to see who might have created them (ignoring any privacy issues.)

**Reply by radh21301** [reply]
Ah. I see. 

To make it simple, maybe the storage label can be renamed to user label - and this is used across both the 'storage template' as well as 'upload' folder.

**Reply by deddc23efb** [reply]
That's actually what I expected to see and I was surprised that the storage label wasn't applied to the uploads as well.  That would work well too. I've discovered that the mapping I want IS actually shown in the Immich-cli-command list-users. Would still be nice to see the same details in the Web Admin view as the the immich-cli-command.

**Comment by bo0tzz** [comment]
This is implemented these days; the user view under the admin panel shows the UUID
